config/core/test-configs-chromeos.yaml

```diff
@@ -790,21 +790,13 @@ device_types:
         modules: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/chromiumos-sarien/20231106.0/amd64/modules.tar.xz'
       block_device: detect
 
+  hp-14-db0003na-grunt_chromeos:
```

Comment:
Same comment, moving it after `hp-11A-G6-EE-grunt_chromeos` would keep the list alphabetically sorted.

Reply:
Ah, stupid linter, yes, it must consider `14` is a number and `11A` isn't, therefore it puts the former in 1st position... Not a big problem, so yeah, let's follow the linter recommendation, however stupid it looks to us ;)